openSUSE/open-build-service
# Bring back Report Bug action

Fixes #14359

## How to test

* `osc api -e /configuration` and paste `<bugzilla_url>https://bugzilla.opensuse.org</bugzilla_url>`
* go to a project and make a user bugowner
* left side navigation should on `ProjectController#show` include a "Report Bug" link now
* it should also be there if you view this page in a private browser window

![Screenshot from 2023-05-23 11-39-05](https://github.com/openSUSE/open-build-service/assets/514785/5e26335f-3341-4bdc-aab9-f72e6dae36c4)
